live-site/components/welcome-section/team-race/Team.styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import { max } from '../../../../shared-ui/lib/responsive';
+
+const StyledTeamContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  @media ${max.tablet} {
+    flex-direction: column;
+  }
+`;
+
+const StyledTeamHeader = styled(H3)`
+  color: ${colors.WHITE};
+`;
+
+const StyledTeamName = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.BUTTON_GREEN};
+  font-size: 1.5em;
+`;
+
+const StyledTeamPoint = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.RACE};
+  font-size: 1.5em;
```

Comment:
Same as above mentioned - add different breakpoints to make sure the font is always small enough so that everyhting is on one line